### DISCUSSION TITLE
Car design - reliability

### DISCUSSION THREAD

**Discussion by felipedemelobarbosa** [domhnallmorr/Pitwall_Flet#6] [Ideas]
Hello.

I read on your roadmap that you plan to set the reliability rating the same for all teams. However, I believe it would more challenging and not so much more difficult to code if the reliability rating depends on car design and on the track raced.

Also, I have some ideas regarding car design so it could be depend on other factors than team facility rating and chief designer rating.

**Comment by domhnallmorr** [comment]
Thank you.

So for V1, I really want to keep the game very simple. Currently the race simulation has a 20% chance of retiring for all drivers (see particpant_model.py, calculate_if_retires method).

Once V1 is complete I will start thinking about adding more complexity as you suggest.

Please feel free to write down any ideas here. Thanks.

**Comment by felipedemelobarbosa** [comment]
Hello.

Thank you for answering.

Did you see my commit regarding drivers.txt?

Regards,

Felipe de Melo Barbosa


Em qua., 25 de set. de 2024 às 17:04, Domhnall Morrissey <
***@***.***> escreveu:
